source/manual/replicate-app-data-locally.html.md

```diff
@@ -126,7 +126,13 @@ Then follow the instructions above for importing using the `-s` flag.
 
 ## If you're running out of disk space
 
-See [running out of disk space in development](/manual/development-disk-space.html).
+See [troubleshooting Vagrant](/manual/troubleshooting-vagrant.html).
```

Comment:
I think we split up `troubleshooting-vagrant.html` because it became too large (https://github.com/alphagov/govuk-developer-docs/pull/462) so I wonder if it's better to create a new "VirtualBox installation fails on High Sierra" page. 

We should probably redirect `/manual/troubleshooting-vagrant.html` somewhere appropriate. Where was it linked from?

Reply:
It was only linked from `get-started`, so it was a bit irrelevant as people probably won't run out of space while first installing everything.